src/Symfony/Bridge/Monolog/Handler/ServerLogHandler.php

```diff
@@ -105,7 +105,7 @@ private function doWrite(array|LogRecord $record): void
     {
         $recordFormatted = $this->formatRecord($record);
 
-        set_error_handler(self::class.'::nullErrorHandler');
+        set_error_handler(fn () => null);
```

Comment:
maybe we have more of these in the codebase? can you have a look?

Reply:
Done. I have update some, and left others. We have some [check](https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php#L89) in the code base with the array syntax. So it's not trivial to update everything. 

I updated only the simple cases